File: westcoast/src/main/java/frc/commands/motors/SetTwoMotors.java
package frc.commands.motors;

import edu.wpi.first.wpilibj.SpeedController;
import edu.wpi.first.wpilibj2.command.CommandBase;

public class SetTwoMotors extends CommandBase {
    private final SpeedController mainMotor;
    private final SpeedController reversibleMotor;
    private final double speed;
    private final boolean inverted;

    public SetTwoMotors(
            SpeedController mainMotor,
            SpeedController reversibleMotor,
            double speed,
            boolean isInverted
    ) {
        this.mainMotor = mainMotor;
        this.reversibleMotor = reversibleMotor;
        this.speed = speed;
        this.inverted = isInverted;
    }

    public SetTwoMotors(
            SpeedController mainMotor,
            SpeedController secondMotor,
            double speed
    ) {
        this(mainMotor, secondMotor, speed, false);
    }

    @Override
    public void initialize() {
        reversibleMotor.setInverted(inverted);
    }

    @Override
    public void execute() {
        mainMotor.set(speed);
        reversibleMotor.set(speed);
    }

    @Override
    public void end(boolean interrupted) {
        mainMotor.stopMotor();
        reversibleMotor.stopMotor();
    }

    @Override
    public boolean isFinished() {
        return false;
    }
}

File: westcoast/src/main/kotlin/frc/commands/motors/SetTwoMotors.kt
package frc.commands.motors

import edu.wpi.first.wpilibj.SpeedController
import edu.wpi.first.wpilibj2.command.CommandBase

class SetTwoMotors(
    private val mainMotor: SpeedController,
    private val reversibleMotor: SpeedController,
    private val speed: Double,
    private val inverted: Boolean = false
) : CommandBase() {

  override fun initialize() {
    reversibleMotor.inverted = inverted
  }

  override fun execute() {
    mainMotor.set(speed)
    reversibleMotor.set(speed)
  }

  override fun end(interrupted: Boolean) {
    mainMotor.stopMotor()
    reversibleMotor.stopMotor()
  }

  override fun isFinished(): Boolean {
    return false
  }
}